backend/src/helpers/auth.js

```diff
@@ -59,8 +61,38 @@ const verifyToken = (token, jwksUri = null) => {
         }
     });
 }
+
+/**
+ * Verify if the user already has an entry in the database.
+ * If not, create one.
+ * 
+ * @param {Object} token 
+ * @returns {Promise}
+ */
+const verifyUserExists = (token) => {
+    return new Promise((resolve, reject) => {
+        const decodedToken = jwt.decode(token, { complete: true });
+        if (decodedToken?.payload && decodedToken?.payload?.email) {
+            const userPayload = decodedToken?.payload;
+            userModel.findByEmail(userPayload.email)
```

Comment:
shawn and i had concerns about accessing the model from here, but this is fine until we add more structure later that is not present right now. this note is just a reminder for us (or just me).

Reply:
I'm not seeing the issue with accessing the model this way. Can one of you explain the problem when you get a moment?